### ISSUE TITLE
Issues with Unreal GSDK plugin

### ISSUE THREAD

**Issue by dgkanatsios** [PlayFab/gsdk#85]
copying from https://community.playfab.com/idea/55633/the-playfabgsdk-github-tutorial-should-be-updated.html

FYI @valexao @stkrwork 

> I am working with 4.26.2-Chaos unreal build and I downloaded the GDSK-master for it, the folder that the plugin is in is called UnrealPlugin. Either that folder name should be changed to PlayfabGSDK, or the portion of the tutorial that says
> 
> 'First, open your File Explorer and create a folder called “Plugins” in your games' root directory. Then drag the Playfab GSDK plugin file into the Plugins folder.'
> 
> should be updated to warn that simply putting the file in there will not work when trying to build it. It will say it cannot find the plugin. It took me a while to figure out there needs to be another folder that the plugin resides in (example: Plugins>PlayfabGSDK>PlayfabGSDK.uplugin)
> 
> Additionally, I kept getting an error when attempting to generate the files for my project: "Couldn't set association for project. Please make sure the file is writeable". I found out when I was copying and pasting from this example in the tutorial (the portion where you edit your .uproject file) that the spacing needs to be exactly four spaces ahead of the previous line. I had attempted to copy and paste everything on and below line in VS2019:
> 
> "AdditionalDependencies": [
> I guess it was adding in some additional space or had less, I do not know since I am not experienced with c++. I just deleted all space before "Engine" until it lined with the beginning of "AdditionalDependencies", then hit space four times. I put an example of the additional comment that might help. The reason I didn't copy and paste the entire thing was because my EngineAssociation was a long and difficult to remember value so I just wanted to add what wasn't present. Hope this helps, it took me two days and several re-installs due to this confusion.

```
{
    "FileVersion": 3,
    "EngineAssociation": "4.26",
    "Category": "",
    "Description": "",
    "Modules": [
        {
            "Name": "<projectname>",
            "Type": "Runtime",
            "LoadingPhase": "Default",
            "AdditionalDependencies": [
                "Engine"		//four spaces before engine
            ]
        }
    ],
    "Plugins": [                    // Add this if it doesn't exist
        {                           // Add this
            "Name": "PlayfabGSDK",  // Add this
            "Enabled": true         // Add this
        }                           // Add this
    ]                               // Add this if it doesn't exist
}
```

**Comment by chris-gong** [comment]
I believe the second part of the issue regarding the four spaces before the word "Engine" was caused by the user since the uproject file snippet in the unreal plugin does indeed have four spaces from where the indentation of the previous line starts and the word "Engine". Just tested this out by copying and pasting the entire snippet from the readme, removing the comments, and replacing the values and was able to generate project files and build the project fine. However, I think that most people will just add in the "Plugins" portion and leave the rest alone.

**Comment by dgkanatsios** [comment]
I believe that this was fixed by #86 so I'll close this and reopen if the issue persists. Thanks all!
